src/posts/gpu-pipeline/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Accelerating AI/ML Workflows in Earth Sciences with GPU-Native Xarray and Zarr (and more!)'
+date: '2025-05-01'
+
+authors:
+  - name: Negin Sobhani
+    github: negin513
+  - name: Wei Ji Leong
+    github: weiji14
+  - name: Max Jones
+    github: maxjones
+  - name: Akshay Subranian
+    github: akshaysubr
+  - name: Thomas Augspurger
+    github: tomaugspurger
+  - name: Katelyn Fitzgerald
+    github: kafitzgerald
+
+summary: 'How to accelerate AI/ML workflows in Earth Sciences with GPU-native Xarray and Zarr.'
+---
+
+# Accelerating AI/ML Workflows in Earth Sciences with GPU-Native Xarray and Zarr (and more!)
+
+## Introduction
+
+In large-scale geospatial AI and machine learning workflows, data loading is often the main bottleneck. Traditional pipelines rely on CPUs to preprocess and transfer massive datasets from storage to GPU memory, consuming resources and limiting scalability and effective use of GPU resources.
+
+To tackle this issue, a team from the [National Center for Atmospheric Research (NSF-NCAR)](https://ncar.ucar.edu) and [Development Seed](https://developmentseed.org) with mentors from [NVIDIA](https://www.nvidia.com) participated in a GPU hackathon to demonstrate how AI/ML workflows in Earth system sciences can benefit from GPU-native workflows using tools such as [Zarr](https://zarr.readthedocs.io/), [KvikIO](https://docs.rapids.ai/api/kvikio/stable/), and [DALI](https://developer.nvidia.com/dali).
```

Comment:
Some institution logos here could be nice visually

Reply:
Good Idea! Added!

Reply:
BTW, you meant the logo of NCAR, devseed, etc? That is added now at the end of the post, but I am open to bring them higher. 

I can also include the logos of packages( i.e. cupy, etc.) but not sure if that was what you meant.